vendor/assets/stylesheets/rolodex/settings/utilities/_layout.sass

```diff
@@ -0,0 +1,51 @@
+// Layout
+// ========================================
+
+// Floats
+
+.pull-left
+  float: left
+
+.pull-right
+  float: right
+
+// Clearfix
+
+.group
+  +group
+
+// Grid
+// ========================================
+
+.grid
+  list-style: none
+  +rem(letter-spacing, -4px)
+
+.opera:-o-prefocus,
```

Comment:
Interesting. Does the`.class` select mean this will only work on elements with that class? Or does opera just ignore the base selector, and only pay attention to the pseudo selector?